### DISCUSSION TITLE
Referencing models

### DISCUSSION THREAD

**Discussion by JimmyTron** [SpaceyaTech/Team-Rio-Django#55] [Q&A]
How best would we reference the  models?
![Screenshot_2022-11-29-12-32-49-141_com github android](https://user-images.githubusercontent.com/23496280/204494603-4e91de1f-7e7c-4af2-937f-a4163433c6fb.png)

- [x] relative reference
- [x] absolute reference

**Comment by AmbroseOtundo** [comment]
I would love the absolute import. It gives a clear sense of how things are. If later on our application becomes complex and verbose then we can use relative. My thoughts though.

**Comment by Eva-Wamuyu** [comment]
I support absolute referencing here... and what about this:
from  accounts import models

**Reply by JimmyTron** [reply]
Does the same and is even more clear

**Reply by JosephThuku** [reply]
i think also that is better and even gives one  good idea of which app models you are using
